hack/builder/version.sh

```diff
@@ -1,3 +1,3 @@
 VERSION=$(date +"%y%m%d%H%M")-$(git rev-parse --short HEAD)
 # TODO: reenable ppc64le when new builds are available
-ARCHITECTURES="amd64 arm64v8"
```

Comment:
`ARCHITECTURES="amd64 arm64"`

Reply:
Also the change file `hack/builder/build.sh`, "arm64v8" to "arm64".

Reply:
Good catch!